python-packages/fle_utils/handlebars/loader.py

```diff
@@ -49,12 +53,18 @@ def get_template_module_paths(module_name):
 
     return paths
 
+
 def load_template_sources(module_name):
 
     templates = []
 
+    if is_windows():
+        slash_character = "\\"
+    else:
+        slash_character = "/"
+
     for filepath in get_template_module_paths(module_name):
-        template_name = filepath.split("/")[-1].split(".handlebars")[0]
+        template_name = filepath.split(slash_character)[-1].split(".handlebars")[0]
```

Comment:
There should be much nicer `os.path` ways to do this, rather than manually splitting.